CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 # Changelog
 
+## [1.13.0](https://github.com/openeuropa/oe_paragraphs/tree/1.13.0) (2022-04-07)
+[Full Changelog](https://github.com/openeuropa/oe_paragraphs/compare/1.12.0...1.13.0)
```

Comment:
```suggestion
## [1.13.0](https://github.com/openeuropa/oe_paragraphs/tree/1.13.0) (2022-04-07)

[Full Changelog](https://github.com/openeuropa/oe_paragraphs/compare/1.12.0...1.13.0)
```

Reply:
We have talked with @sergepavle and agreed that this is not needed given generator doesn't add additional line.